docs/changelog-fragments.d/236.bugfix.md

```diff
@@ -0,0 +1,4 @@
+Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
```

Comment:
Don't forget to review for the use of the past tense references. Changelog shouldn't look like a TODO-list.